spec/graphql/mutations/users/update_user_details_mutation_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::Users::UpdateUserDetailsMutation do
+  let(:user) { create :user }
+  let(:organization) { create :organization }
+  it 'updates a user\'s github org' do
+    mutation = <<-GRAPHQL
+			mutation {
```

Comment:
Can we downsize the spacing in here down to 2 spaces?

Reply:
I was not having good luck with my heredocs with 2 spaces for some reason